### ISSUE TITLE
[BUG] Missing Symbol in Cpp backend

### ISSUE THREAD

**Issue by LDenninger** [masadcv/FastGeodis#59]
First of all, thanks for the great work! I have been looking for something like that for quite some time.

**Describe the bug**
Unfortunately, I do not manage to to get the library running as I get the error of an undefined symbol:
```
>>> import FastGeodis
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/denninger/anaconda3/envs/nimbro_soccer/lib/python3.10/site-packages/FastGeodis/__init__.py", line 34, in <module>
    import FastGeodisCpp
ImportError: /home/denninger/anaconda3/envs/nimbro_soccer/lib/python3.10/site-packages/FastGeodisCpp.cpython-310-x86_64-linux-gnu.so: undefined symbol: _ZN2at4_ops14conv2d_padding4callERKNS_6TensorES4_RKSt8optionalIS2_EN3c108ArrayRefINS9_6SymIntEEENS9_17basic_string_viewIcEESC_SB_

```

**To Reproduce**
I have installed the library via Pip and through the release published on GitHub. Both give the same error when trying to import FastGeodis

**Desktop:**
 - OS: Ubuntu 20.4.6
 - Python: 3.10.3
 - Virtual Environemnt: Conda
 - PyTorch: 2.1.2
 - Cuda-Toolkit: 12.1

Maybe somebody sees what I did wrong and can help me to fix this issue!
Greetings, Luis

**Comment by masadcv** [comment]
Hi Luis, 
It seems there is some issue with your installation. 
Can you please provide the commands/steps you did to install the library?

**Comment by masadcv** [comment]
Additionally, for conda can you try the following steps for installing this?
https://github.com/masadcv/FastGeodis/issues/6#issuecomment-1166380502

**Comment by LDenninger** [comment]
I tried installing it using: `pip install FastGeodis` and `pip install FastGeodis --no-build-isolation`. Both had the same problem. But I installed PyTorch via conda previously which seems to be the issue.  It seems the Torch binaries are not found/included by FastGeodis when doing it this way.

I managed to get it working when installing PyTorch through pip as well.

Now everything seems to work fine!

Thanks for the help and work again!
